win32/devdoc/gballoc_ll_jemalloc.md

```diff
@@ -165,4 +167,28 @@ static void jemalloc_print_stats_callback(void* context, const char* text)
 
 **SRS_GBALLOC_LL_JEMALLOC_01_009: [** If `text` is NULL, `jemalloc_print_stats_callback` shall return. **]**
 
-**SRS_GBALLOC_LL_JEMALLOC_01_010: [** Otherwise, `jemalloc_print_stats_callback` shall print (log) `text`, breaking it does in chunks of `LOG_MAX_MESSAGE_LENGTH / 2`. **]**
\ No newline at end of file
+**SRS_GBALLOC_LL_JEMALLOC_01_010: [** Otherwise, `jemalloc_print_stats_callback` shall print (log) `text`, breaking it does in chunks of `LOG_MAX_MESSAGE_LENGTH / 2`. **]**
+
+### gballoc_ll_set_decay
+
+```c
+MOCKABLE_FUNCTION(, int, gballoc_ll_set_decay, int64_t, decay_milliseconds);
```

Comment:
>decay_milliseconds [](http://example.com/codeflow?start=56&length=18)

Is -15 a valid argument value here?

Reply:
I have added a check to fail if the passed decay value is less than -1.